README.md

```diff
@@ -205,17 +205,23 @@ and by cryptographers to
 
 Cryptol has been under development for over a decade with many people
 contributing to its design and implementation. Those people include (but
-are not limited to) Iavor Diatchki, Aaron Tomb, Adam Wick, Brian
-Huffman, Dylan McNamee, Joe Kiniry, John Launchbury, Matt Sottile, Adam
-Foltzer, Joe Hendrix, Trevor Elliott, Lee Pike, Mark Tullsen, Levent
-Erkök, David Lazar, Joel Stanley, Jeff Lewis, Andy Gill, Edward Yang,
-Ledah Casburn, Jim Teisher, Sigbjørn Finne, Mark Shields, Philip Weaver,
-Magnus Carlsson, Fergus Henderson, Joe Hurd, Thomas Nordin, John
-Matthews and Sally Browning. In addition, much of the work on Cryptol
-has been funded by, and lots of design input was provided by the team at
-the [NSA Laboratory for Advanced Cybersecurity
-Research](https://www.nsa.gov/what-we-do/research/cybersecurity-research/),
-including Brad Martin, Frank Taylor, and Sean Weaver.
+are not limited to) Aaron Tomb, Adam Foltzer, Adam Wick, Alexander
+Bakst, Andrew Kent, Andrey Chudnov, Andy Gill, Benjamin Barenblat, Ben
+Jones, Ben Selfridge, Brian Huffman, Brian Ledger, David Thrane
+Christiansen, David Lazar, Dylan McNamee, Eddy Westbrook, Edward Yang,
+Eric Mertens, Eric Mullen, Fergus Henderson, Iavor Diatchki, Jeff Lewis,
+Jim Teisher, Joe Hendrix, Joe Hurd, Joe Kiniry, Joel Stanley, Joey
+Dodds, John Launchbury, John Matthews, Jonathan Daugherty, Kenneth
+Foner, Kevin Quick, Kyle Carter, Ledah Casburn, Lee Pike, Levent Erkök,
+Lisanna Dettwyler, Magnus Carlsson, Mark Shields, Mark Tullsen, Matt
+Sottile, Nathan Collins, Philip Weaver, Robert Dockins, Sally Browning,
+Sam Anklesaria, Sigbjørn Finne, Stephen Magill, Thomas Nordin, Trevor
+Elliott, and Tristan Ravitch.
+
+Much of the work on Cryptol has been funded by, and lots of design input
+was provided by the team at the [NSA's Trusted Systems Research
+Group](http://www.nsa.gov/research/ia_research/), including Brad Martin,
```

Comment:
Also, the affiliation changed. It was correct as Laboratory for Advanced Cyber Security Research.

Reply:
Undone back to the right one.